src/patternfly/components/Label/examples/label-table-example.hbs

```diff
@@ -0,0 +1,206 @@
+<!-- Table One -->
+{{#> table table--grid="true" table--modifier="pf-m-grid-md" table--attribute='aria-label="This is a simple table example"'}}
```

Comment:
I'd say a table shouldn't be included here, we could link to a demo in the description to show a label being used in a table. Generally we don't want to include anything but the component in the component examples.

Reply:
I was just going to leave it here for the review. Do you think they could go inside the table component? (Maybe I could create an issue to add it in after this PR is merged)

Reply:
Should we remove this file since it isn't being used anymore, or do you want to keep it for use later or in some other context?